concordia/settings.py

```diff
@@ -0,0 +1,265 @@
+# TODO: use correct copyright header
```

Comment:
This seems to conflict with the settings updates which @rstorey recently made and since there's no comment on the commit I can't tell whether that's intentional